src/filefuncs.c

```diff
@@ -732,8 +731,17 @@ Bool RemoveFiles(const char *fspec, LOG_INFO *LogInfo) {
         for (i = 0; i < nfiles; i++) {
             fNamePlusDLen = fname + dlen;
             (void) sw_memccpy_inc(
-                (void **) &fNamePlusDLen, (void *) flist[i], '\0', &writeSize
+                (void **) &fNamePlusDLen,
+                endFnamePtr,
+                (void *) flist[i],
+                '\0',
+                &writeSize
             );
+            if (bufferFull) {

```

Comment:
RemoveFiles(): calls `(void) sw_memccpy_inc(...);` but then checks `if (bufferFull) {...}` -- shouldn't this be instead  `bufferFull = sw_memccpy_inc(...); if (bufferFull) {..}`?